samtranslator/validator/validator.py

```diff
@@ -7,49 +9,254 @@
 
 
 class SamTemplateValidator(object):
+    """
+    SAM template validator
+    """
+
+    # Useful to find a unicode prefixed string type to replace it with the non unicode version
+    # On Python2, the validator returns types in error messages prefixed with 'u'
+    # We remove them to be consistent between python versions
+    # Example: "u'integer'" -> "'integer'"
```

Comment:
@hawflau Could be worth a look here. I know you are looking into Python2 removal stuff, so want to flag this to you. Not sure if this will be an issue at all with some of the hashing work you are doing between python2 and python3.

Reply:
It seems this is only used to remove the prefix `u` in error messages. The error messages are not used as part of the hash generation (both directly and indirectly), so I don't think it will cause any issue.

We will not need this once we completely remove support for Python2. For now, it makes the error messages generated in a Python2 runtime and a Python3 runtime consistent.